ios/RNCPushNotificationIOS.m

```diff
@@ -125,6 +125,20 @@ @implementation RNCPushNotificationIOS
   return formattedNotification;
 }
 
+API_AVAILABLE(ios(10.0))
+static NSDictionary *RCTFormatOpenedUNNotification(UNNotification *notification)
+{
+  NSMutableDictionary *formattedNotification = [RCTFormatUNNotification(notification) mutableCopy];
+  UNNotificationContent *content = notification.request.content;
+    
+  NSMutableDictionary *userInfo = [content.userInfo mutableCopy];
+  userInfo[@"userInteraction"] = @"1";
```

Comment:
Using a number would make this compliant with the remote userInteraction value.
```obj-c
userInfo[@"userInteraction"] = [NSNumber numberWithInt:1];
```

Reply:
Thanks! Good catch.